DotNet/Web.config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0"?>
+<?xml version="1.1.1beta"?>
```

Comment:
This is referring to the [version of XML](https://msdn.microsoft.com/en-us/library/ms256048(v=vs.110).aspx) standard that this file conforms to, not the version of the proxy.  I think we should stick with 1.0 here.